bot/exts/info/information.py

```diff
@@ -420,13 +420,8 @@ async def user_messages(self, user: MemberOrUser) -> Tuple[Union[bool, str], Tup
             if e.status == 404:
                 activity_output = "No activity"
```

Comment:
Uh oh, because of the code below it won't this try to iterate the string?
```python
            activity_output = "\n".join(
                f"{name}: {metric}" for name, metric in zip(["Messages", "Activity blocks"], activity_output)
            )
```

Shouldn't it be `activity_output = ["No activity"]`? Have we never hit this codepath?

Reply:
??? that .join is within the else block, so will never get hit if this line is.